Frameworks/UIKit/UIWebView.mm

```diff
@@ -251,6 +251,25 @@ - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
 }
 
 /**
+ @Status Caveat
+  @Notes This is a workaround. Original UIWebView does not have this method
+*/
+- (void)evaluateJavaScript:(NSString*)javaScriptString completionHandler:(void (^)(id, NSError*))completionHandler {
```

Comment:
Rather than adding a new function, you can do this in stringByEvaluatingJavaScriptFromString.